RandomData.Api.Tests/StringGenerationTests/StringGenerationServiceHelpersTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.IO;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using RandomData.Api.Services.FileReader;
+using RandomData.Api.Services.Random;
+using RandomData.Api.StringGeneration;
+using RandomData.Api.StringGeneration.ServiceImplementations;
+using RandomData.Api.Tests.Services.FileReaderService;
+using Xunit;
+
+namespace RandomData.Api.Tests.StringGenerationTests
+{
+    public class StringGenerationServiceHelpersTests
+    {
+        [Fact]
+        public void IsAddStringGenerationServicesWorkingProperly()
+        {
+            //arrange
+            var serviceCollection = new ServiceCollection();
+            var expectedOptions = new StringGenerationServiceHelpers.StringGenerationServiceOptions
+            {
+                WordsDictionaryLocation = "words.json"
+            };
+            IConfiguration configuration = new ConfigurationBuilder()
+                .AddJsonStream(
```

Comment:
I think that content of file doesn't matter during initialization. You can add empty stream.

Reply:
Currently it does matter because the services depend on correct json config. Without it the exception is thrown.